deployment/crds/quoteattestations.tcs.intel.com.yaml

```diff
@@ -114,6 +114,115 @@ spec:
             type: object
         type: object
     served: true
+    storage: false
+    subresources:
+      status: {}
+  - name: v1alpha2
+    schema:
+      openAPIV3Schema:
+        description: QuoteAttestation is the Schema for the quote attestation API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: QuoteAttestationSpec defines the desired state of QuoteAttestation
+            properties:
+              nonce:
+                description: Nonce base64-encoded nonce used for generating the SGX
+                  Quote. This is required for verifying the provided SGX quote by
+                  the key server(s).
+                format: byte
+                type: string
+              publicKey:
+                description: PublicKey for encrypting the secret, hash is part of
+                  the quote data, base-64 encoded.
+                format: byte
+                type: string
+              quote:
+                description: Quote to be verified, base64-encoded.
+                format: byte
+                type: string
+              quoteVersion:
+                description: QuoteVersion used to for generated quote, default is
+                  ECDSA quote "3"
+                type: string
+              secretName:
+                description: SecretName is name of the Secret object (in the same
+                  namespace) to keep the wrapped on secrets (only needed for KeyProvisioning
+                  request type) which is an opeque type. The secret data must contain
+                  two map elements `tls.key` and `tls.cert` and the values are the
+                  base64 encoded encrypted CA key and base64 encoded x509(PEM encoded)
+                  certificate. This must be added only after a successful quote validation
+                  and before updating the status condition.
+                type: string
+              serviceId:
+                description: // ServiceID holds the unique identifier(name?) that
```

Comment:
Why `//`?

Reply:
This has been a typo in the field comments with an extra '//' prefix since the v1alpha1 API. I will fix it now.

Reply:
Fixed.